tests/vm_test.go

```diff
@@ -1921,6 +1921,56 @@ status:
 			}, 2*time.Minute, 1*time.Second).Should(Succeed())
 		})
 	})
+
+	Context("[Serial] when node becomes unhealthy", Serial, func() {
+		var nodeName string
+
+		AfterEach(func() {
+			libnode.Untaint(nodeName, libnode.UnschedulableTaint)
+		})
+
+		It("[Serial] the VMs running in that node should be respawned", func() {
+			By("Starting VM")
+			// We need to set RunningStrategy to either Always or ReRunOnFailure, the startVM function sets
+			// the RunningStrategy to Always.
```

Comment:
I'd drop this comment, we want a running VM, rest doesn't matter.

Reply:
Done.